### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by LandonSchropp** [rhysd/conflict-marker.vim#12]
Thanks for making this plugin! This is a huge upgrade over what I was doing before.

**Comment by ranelpadon** [comment]
Yeah, I really want to say thanks for this awesome plugin, just the one I really needed!!! Coming from Atom/VS Code, these editors implemented it elegantly, but JetBrains refused to implement this inline-conflict resolution workflow in their IDEs. I filed an StackOverflow issue/ticket to JetBrains, but they said they have no plans to implement it:
https://stackoverflow.com/questions/63593048/inline-merge-conflict-resolver-in-pycharm-intellij-jetbrains-ides#answer-63703195

 So, quite surprised that there's a similar plugin for Vim! I checked other Vim-conflicts plugins also but they're not really what I wanted. Cheers. :)
